blob/helper.go

```diff
@@ -17,16 +17,16 @@ func SharesToBlobs(rawShares []share.Share) ([]*Blob, error) {
 		return nil, ErrBlobNotFound
 	}
 
-	appShares := make([]shares.Share, 0, len(rawShares))
-	for _, shr := range rawShares {
-		bShare, err := shares.NewShare(shr)
-		if err != nil {
-			return nil, err
-		}
-		appShares = append(appShares, *bShare)
+	appShares, err := toAppShares(rawShares...)
+	if err != nil {
+		return nil, err
 	}
+	return parseShares(appShares)
+}
 
-	shareSequences, err := shares.ParseShares(appShares, true)
+// parseShares takes shares and converts them to the []*Blob.
+func parseShares(appShrs []shares.Share) ([]*Blob, error) {
```

Comment:
what's the difference between this and `createBlobs` ?

Reply:
renamed `createBlobs`  -> `buildBlobsIfExist`